config/Shared/config_default.php

```diff
@@ -335,6 +336,9 @@
 $config[PropelConstants::ZED_DB_REPLICAS] = json_decode(getenv('SPRYKER_DB_REPLICAS') ?: '[]', true);
 $config[PropelConstants::USE_SUDO_TO_MANAGE_DATABASE] = false;
 
+// >>> DATABASE REPLICA CACHE
+$config[PropelReplicationCacheConstants::CACHE_TTL] = 2;
```

Comment:
Shouldn't we also integrate 
```
$config[PropelReplicationCacheConstants::IS_REPLICATION_ENABLED] = (bool)$config[PropelConstants::ZED_DB_REPLICAS];
```

From https://github.com/spryker/suite-nonsplit/pull/6379/files

Reply:
It was, but we agreed the replication has to be disabled by default and I removed it. 
Please, check https://spryker.slack.com/archives/C03KPK97UH3/p1683029569719389